src/widgets-service.js

```diff
@@ -36,7 +36,8 @@ WidgetsService.prototype.newCategoryModel = function (attrs, layer) {
   var dataviewModel = this._dataviews.createCategoryModel(layer, {
     type: 'category',
     column: attrs.column,
```

Comment:
This is a good place to ask: what is the difference between `column` and `aggregationColumn` ?